src/schema/v2/homeView/sections/DiscoverSomethingNew.ts

```diff
@@ -0,0 +1,46 @@
+import { ContextModule } from "@artsy/cohesion"
+import { HomeViewSection } from "."
+import { HomeViewSectionTypeNames } from "../HomeViewSection"
+import { connectionFromArray } from "graphql-relay"
+
+const marketingCollectionSlugs = [
+  "figurative-art",
+  "new-from-leading-galleries",
+  "paintings",
+  "prints",
+  "street-art",
+  "black-and-white",
+  "art-under-1000",
+  "art-for-small-spaces",
+  "cool-toned-artworks",
+]
+
+export const DiscoverSomethingNew: HomeViewSection = {
+  id: "home-view-section-discover-something-new",
+  featureFlag: "diamond_home-view-marketing-collection-categories",
+  contextModule: "" as ContextModule, // TODO: fill in with real value
+  type: HomeViewSectionTypeNames.HomeViewSectionCards,
+  component: {
+    title: "Discover Something New",
+  },
+  requiresAuthentication: false,
+  resolver: async (_parent, args, context, _info) => {
+    const { body } = await context.marketingCollectionsLoader({
+      slugs: marketingCollectionSlugs,
+    })
+
+    const cards = body.map((marketingCollection) => {
+      return {
+        href: `/collection/${marketingCollection.slug}`,
+        entityID: marketingCollection.id,
```

Comment:
**thought (non-blocking):** Is entityID and entityType the right naming here? Do we need to be more specific? I’m still not quite sure, as this has come up a few times in different discussions.

Reply:
For a section type that renders a list of cards, the default component in Eigen to support the type could defer to an `href` attribute to handle navigation on tap. By also threading through the entity ID/type, clients also have the option of adding this metadata to analytic calls. For example, we often fire an event that includes what the intended destination screen is.

For example, we could have an event with the following structure:

```
{
  action: ActionType.tappedCardGroup,
  context_module: "discoverSomethingNew,
  context_screen_owner_type: OwnerType.home,
  destination_screen_owner_type "MarketingCollection",
  destination_screen_owner_id: "d78e9a17-ccf6-4104-b4e9-95c18f6412df", 
  horizontal_slide_position: index,
  module_height: "double",
  type: "thumbnail",
}
```

You can see other examples of analytics payloads [here in Eigen](https://github.com/artsy/eigen/blob/ea625b401a7bd8f00a7466dc7d326da3fa5c7aee/src/app/Scenes/HomeView/useHomeViewTracking.ts).

Reply:
So far we've avoided using ` top-level `analytics` field in favor of contextModule and ownerType fields. These `entity` fields to me represent a way of exposing what type of entity/item/thing the cards are representing. That _could_ be used for internal navigation, as I think will be the case for the "Explore By" section instance where we aren't providing `href` but it could also be used for internal client implementation details (like analytics).

But I'd rather not enforce this at the schema level especially for a generic "card" section type like this.

Reply:
Otherwise, it seems tricky to remain generic _and_ give Eigen the context/data it needs in order to navigate to the right internal Marketing Collection **category** screen.

Reply:
Thanks for the explanation - entityID and entityType seemed a bit misleading to me at first, but this clarification definitely helps